backend/src/main/java/ca/bc/gov/app/security/ApiAuthorizationCustomizer.java

```diff
@@ -45,22 +45,28 @@ public void customize(AuthorizeExchangeSpec authorize) {
         // IDIR users, BCEID business users, and BCSC users can access the codes endpoint
         .pathMatchers("/api/codes/**")
         .hasAnyRole(
-            ApplicationConstant.ROLE_IDIR_USER,
+            ApplicationConstant.ROLE_VIEWER,
+            ApplicationConstant.ROLE_EDITOR,
+            ApplicationConstant.ROLE_ADMIN,
             ApplicationConstant.ROLE_BCEIDBUSINESS_USER,
             ApplicationConstant.ROLE_BCSC_USER
         )
         // IDIR users, BCEID business users, BCSC users, and service users can access the districts endpoint
         .pathMatchers("/api/districts/**")
         .hasAnyRole(
-            ApplicationConstant.ROLE_IDIR_USER,
+            ApplicationConstant.ROLE_VIEWER,
+            ApplicationConstant.ROLE_EDITOR,
+            ApplicationConstant.ROLE_ADMIN,
             ApplicationConstant.ROLE_BCEIDBUSINESS_USER,
```

Comment:
@paulushcgcj I would suggest renaming the constants ROLE_BCEIDBUSINESS_USER, ROLE_BCSC_USER and ROLE_IDIR_USER as those are not roles, these are authentication methods. I would use something like AUTH_TYPE_IDIR, AUTH_BCEIDBUSINESS, AUTH_BCSC.

Reply:
Renamed. It makes sense to rename it to something else as it can be misleading. This is just a reflex of the old approach